pkg/cryptoauthlib/include/atca_params.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2019 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     pkg_cryptoauthlib cryptoauthlib security crypto
+ * @{
+ *
+ * @file
+ * @brief       Default configuration for Microchip CryptoAuth devices
+ *
+ * @author      Lena Boeckmann <lena.boeckmann@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#ifndef ATCA_PARAMS_H
+#define ATCA_PARAMS_H
+
+#include "cryptoauthlib.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default GPIO pin for wake function.
+ * 
+ * @brief   If the device is asleep it needs to be woken up by driving the SDA pin
```

Comment:
do not use `@name` and `@brief` together, use `@name` for groups and `@brief` otherwise. 

In this case, just omit `@brief` but leave the text as is for detailed info